events/wc_crisis_events.txt

```diff
@@ -0,0 +1,758 @@
+				trait=zealous trait=sturdy trait=brave trait=wroth trait=being_undead
+				age = 400
+			}
+			
+			new_character = {
+				set_graphical_culture = vrykul_undead
+			}
+			
+			create_character = {
+				name = Saor
+				dynasty = 70204
+				culture = drust religion = throsic
+				attributes = {
+					martial=5 diplomacy=5 stewardship=4 intrigue=4 learning=6
+				}
+				trait=grey_eminence
+				trait=creature_vrykul
+				trait=class_druid_5
+				trait=zealous trait=shrewd trait=cruel trait=temperate trait=patient trait=being_undead
+				age = 400
+				add_trait_undead_effect = yes
+			}
+			
+			new_character = {
+				set_graphical_culture = vrykul_undead
+			}
+		}
+		# Make the person that fired the decison a vassal of Gorak Tul
+		event_target:target_summoner_of_gorak = {
+			if = {
+				limit = {
+					character_can_be_vassalized_trigger = yes
+				}
+				set_defacto_liege = event_target:target_gorak_tul
+			}
+		}
+		# Declares war against Kul Tiras (wip)
+		k_kul_tiras = {
+			holder_scope = {
+				if = {
+					limit = {
+						dark_public_religion_trigger = no
+					}
+					reverse_unsafe_war = {
+                        target = event_target:target_gorak_tul
+                        casus_belli = invasion
+                        thirdparty_title = PREV
+                    }
+				}
+            }
+        }
+		
+		# The Thornspeakers take action
+		73 = { province_event = { id = WCCRS.149 days = 7 } }
+	}
+
+	option = {
+		name = EVTOPTA_WCCRS_130
+	}
+}
+
+# The Drust sack Boralus
+narrative_event = {
+	id = WCCRS.140
+	title = "EVTTITLE_WCCRS_140"
+	desc = {
+		trigger = { NOT = { religion = throsic } }
+		text = EVTDESC_WCCRS_140
+	}
+	desc = {
+		trigger = { religion = throsic }
+		text = EVTDESC_WCCRS_141
+	}
+	picture = "GFX_evt_boralus_siege"
+	border = "GFX_event_narrative_frame_war"
+
+	is_triggered_only = yes
+	
+	major = yes
+	major_trigger = {
+		ai = no
+	}
+	show_root = yes
+	hide_from = yes
+	
+	religion = throsic
+
+	trigger = {
+		has_global_flag = drust_incursion_happened
+		NOT = { has_global_flag = drust_sack_boralus }
+		
+		religion = throsic
+		
+		FROM = {
+			title = b_boralus
+			location = {
+				NOT = { has_province_modifier = undead_province }	# I think it is not necessary to explain why I added these ones
+				dark_public_religion_trigger = no
+				culture_group = human_group
+			}
+		}
+	}
+
+	immediate = {
+		set_global_flag = drust_sack_boralus
+		
+		FROM = {
+			location = {
+				add_max_depopulation_effect = yes
+				hidden_effect = {
+					any_province_character = {
+						limit = {
+							NOT = { character = ROOT }
+
+							in_command = no
+						}
+						random = {
+							chance = 75
+							death = {
+								death_reason = death_battle
+								killer = ROOT
+							}
+						}
+					}
+				}
+			}
+		}
+	}
+
+	option = {
+		name = "EVTOPTA_WCCRS_140"
+		trigger = { NOT = { religion = throsic } }
+	}
+	option = {
+		name = "EVTOPTB_WCCRS_140"
+		trigger = { religion = throsic }
```

Comment:
It makes more sense to check true religion. I shouldn't be happy if you're Holy Light in secret.
```suggestion
		trigger = { NOT = { true_religion = throsic } }
	}
	option = {
		name = "EVTOPTB_WCCRS_140"
		trigger = { true_religion = throsic }
```